### PR TITLE
DOP-1337 Add search page with search-results directive

### PR DESCRIPTION
**Pull request by jestapinski**
[JIRA](https://jira.mongodb.org/browse/DOP-1337)
[Spec](https://docs.google.com/document/d/10TAxNOdWDp_vi-D_NTnA_YDQXLeyoPsFa2Banx1Srsw/edit#heading=h.pyglchaeju4o)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/landing/docsworker-xlarge/DOP-1337/search)
[Job](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5f36e7cc2f9a5cfb5b242c80)

This PR adds a new page at `/search` which simply invokes the new `search-results` directive to prepare for the new search UI.

This should not be merged until we are ready for the new Searchbar. Specifically we will also have to merge/deploy:
- [docs-tools #448](https://github.com/mongodb/docs-tools/pull/448)
- A forthcoming Snooty front-end branch `search-ui`

### CODE REVIEW

File: source/search.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+:template: blank
```
Comment (jestapinski) [inline source/search.txt:1]: Perhaps we want to either use this template but bump max-width or just make a new template for search?

Comment (sophstad) [reply source/search.txt:1]: I think a new template makes sense! Perhaps something like `blank-wide`? Whatever you think 😄

**Comment by sophstad** [comment]
Added `do not merge` label as this should not be merged into `master` until we are ready to deploy the new search UI.

File: source/search.txt (hunk 1 above)
Comment (jestapinski) [reply source/search.txt:1]: That's an awesome idea! I made [this ticket](https://jira.mongodb.org/browse/DOP-1371) to track just putting that template into the front-end and will update this PR accordingly.

Comment (jestapinski) [reply source/search.txt:1]: After speaking with Allison, she is okay with just using the blank template we already have. I went ahead and reverted that here

**Pull request merged**